### DISCUSSION TITLE
layui是否有全局配置请求头的功能

### DISCUSSION THREAD

**Discussion by Jcsim0** [layui/layui#2447] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.20

### 问题描述

请问layui是否有全局配置请求头的功能呢？比如使用table时，访问其设置的url时自动加上headers，不需要每次使用table时都要手动添加header参数。

### 业务代码

```auto
无
```

### 浏览器

chrome

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
使用  `table.set()` 设置 header ，或使用 jQuery 的 `$.ajaxSetup()` 做全局 Ajax 设置。
